frontend/src/router/index.js

```diff
@@ -1,112 +1,27 @@
 import Vue from 'vue';
 import Router from 'vue-router';
-import flatMap from 'array.prototype.flatmap';
 
-import HomeView from '../views/HomeView.vue';
-import LoginView from '../views/LoginView.vue';
-import DriveFormView from '../views/DriveFormView.vue';
-import DrivesView from '../views/DrivesView.vue';
-import SuccessfulLogoutView from '../views/SuccessfulLogoutView.vue';
-import PassengerView from '../views/PassengerView.vue';
-import PassengerSubmitView from '../views/PassengerSubmitView.vue';
+import flatMap from 'array.prototype.flatmap';
 
 import store from '../store';
-import * as mutations from '../store/mutations';
-import { deleteStorageData } from '../services/api/auth';
 import { isUserLoggedIn } from '../services/api/user';
 
-Vue.use(Router);
+import routes, {
```

Comment:
```suggestion
import { routes,
```
Or am I missing something?

Reply:
`routes` are the default export, so it will be outside the brackets.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#Description